src/pages/Proxies.tsx

```diff
@@ -201,6 +221,26 @@ export default () => {
                               }}
                             />
                           </Show>
+                          <Show
+                            when={
+                              proxyGroup.icon &&
+                              proxyGroup.icon.startsWith('data:image/svg+xml')
+                            }
+                          >
+                            <div
+                              style={{
+                                'mask-image': `url("${encodeSvg(proxyGroup.icon!)}")`,
+                                '-webkitmask-image': `url("${encodeSvg(proxyGroup.icon!)}")`,
+                                height: `${iconHeight()}px`,
+                                width: `${iconHeight()}px`,
+                                'margin-right': `${iconMarginRight()}px`,
+                                'mask-size': '100% 100%',
+                                'background-color': 'currentColor',
+                                color:
+                                  'var(--icon-p, oklch(var(--p) / var(--tw-bg-opacity)))',
```

Comment:
Where are these css variables come from?

Reply:
the new commit has already deleted the css variables